catalogmanagementv1/catalog_management_v1_examples_test.go

```diff
@@ -377,6 +377,31 @@ var _ = Describe(`CatalogManagementV1 Examples Tests`, func() {
 			offeringID = *offering.ID
 		})
 
+		It(`GetOfferingChangeNotices request example`, func() {
```

Comment:
There are no examples for the get_plan, delete_plan, consumable_plan or set_deprecate_plan operations.